### ISSUE TITLE
Long lines and long Tree View do not show up properly

### ISSUE THREAD

**Issue by astorije** [atom/atom#10543]
When a file gets very long (about 940 characters) or a Tree View has hundreds of file, a glitch prevents further characters or files to show up. They simply are not rendered, leaving the window empty. 

**Reproduction Steps:**

Not sure how to reproduce that issue.
At least load an heavy project (several dozen thousand files), or a file with very long lines (happens on unsaved files too, like pasting a very long unprettified JSON output).
For the It does not happen consistently but it always happens when I start opening subfolders. For the line, it always happens on long lines.

**Screenshots:**

Tree View | Main panel
--- | ---
<img width="236" alt="screen shot 2016-01-21 at 11 15 46" src="https://cloud.githubusercontent.com/assets/113730/12486471/e4894920-c030-11e5-9acd-859f876b5f24.png"> | <img width="578" alt="screen shot 2016-01-21 at 11 20 15" src="https://cloud.githubusercontent.com/assets/113730/12486501/051724be-c031-11e5-9f9d-58ae5d87c52f.png">

Might be 2 different issues but they awfully seem related. Let me know if I should open 2 different issues.

**Environment**

I'm running Atom 1.4.0 on Mac OS X El Capitan but this was appearing in Atom 1.2.x and 1.3.x as well, I was waiting for 1.4.x to be released before opening an issue.
No VM, Shout originally installed via Homebrew Cask and then let it upgrade using Atom's included upgrade mechanism.

**Installed packages:**

```
├── atom-jade@0.3.0
├── atom-jinja2@0.5.0
├── change-case@0.6.0
├── emmet@2.4.0
├── file-icons@1.6.14
├── highlight-selected@0.11.1
├── language-diff@0.4.0
├── language-haml@0.23.15
├── language-nginx@0.6.1
├── language-scala@1.1.4
├── merge-conflicts@1.3.7
├── minimap@4.19.0
├── minimap-find-and-replace@4.4.0
├── minimap-git-diff@4.1.8
├── minimap-highlight-selected@4.3.1
├── minimap-pigments@0.1.7
├── minimap-selection@4.3.1
├── pigments@0.20.0
├── pretty-json@0.4.1
├── project-manager@2.7.4
├── react@0.14.1
├── symbol-gen@1.0.0
└── tab-switcher@1.3.3
```

**Comment by mnquintana** [comment]
Please read the [Reporting Bugs section of the Contributing guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#reporting-bugs) – we need a lot more information in order to help you debug.

**Comment by astorije** [comment]
Hey @mnquintana, I hit Enter while adding the title... Sorry!
Added more information to the issue description.

**Comment by 50Wliu** [comment]
Hey @astorije, thanks for the report!  This is a duplicate of atom/one-dark-ui#109 and has already been fixed in the 1.5.0 betas.

**Comment by astorije** [comment]
Thanks @50Wliu! Sorry I opened a duplicate then, I couldn't find if it was already filed :-/
Can't wait for 1.5 to be publicly released then!
